### DISCUSSION TITLE
41D08670-2DC6-48D5-AB7D-43AD9AAA147F

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#81] [General]
# Exhibit-1

Exhibit-1 also known as EXOI-1, is a fictional solar system discovered by the also fictional Exhibit North Observatory, the confirmation of this system took over 3500 hours of observing time over the span of 10 years. The first transit of the system was detected on January 17. 2011 form Exhibit-1 b, followed by Exhibit-1 d on the 27th and Exhibit-1 c on February 17th, 2011. The gaseous planets were first marked as candidates for radial velocity measurements of the star, later in March of 2012 Exhibit-1 e crossed the face of its star followed by Exhibit-1 f two years later. Observations continued after to affirm the discovery of the outer planets and get more precise measurements, finally, the report confirming the solar system was published in 2021.

https://celestia.mobi/addon?item=41D08670-2DC6-48D5-AB7D-43AD9AAA147F&lang=en

**Comment by exocritic5457** [comment]
f's moons are missing textures and that's pretty much it.
